frontend/src/app/shared/components/storages/storage/storage.component.ts

```diff
@@ -127,6 +136,8 @@ export class StorageComponent extends UntilDestroyedMixin implements OnInit, OnD
       uploadFailed: (fileName:string):string => this.i18n.t('js.storages.file_links.upload_error', { fileName }),
       linkingAfterUploadFailed: (fileName:string, workPackageId:string):string =>
         this.i18n.t('js.storages.file_links.link_uploaded_file_error', { fileName, workPackageId }),
+      draggingManyFiles: (storageType:string):string => this.i18n.t('js.storages.file.dragging_many_files', { storageType }),
```

Comment:
`draggingTooManyFiles`?

Reply:
I see no reason for making it longer. ;) `many` is quite the correct word if you want to emphasize "more the one".